accounts/tests.py

```diff
@@ -1,3 +1,43 @@
-# from django.test import TestCase
+import pytest
+from django.contrib.auth.models import User # noqa
```

Comment:
You can remove the `# noqa` from this line - it's in use later in the tests and I see no flake8 or other errors without it.